test/functionalTest/cases/0000_large_requests/notification_different_sizes.test

```diff
@@ -241,7 +241,7 @@ Date: REGEX(.*)
 1
 ** Notification sizes:
  Sending message 1 to HTTP server: sending message of REGEX((19669|19670|19671|19672|19673|19674|19675)) bytes to HTTP server
- Sending message 2 to HTTP server: sending message of REGEX((21670|21671|21672|21673|21674|21675|21676)) bytes to HTTP server
+ Sending message 2 to HTTP server: sending message of REGEX((21669|21670|21671|21672|21673|21674|21675|21676)) bytes to HTTP server
```

Comment:
Yes, it works. Fixed in 63007a7.

Not that the modification did hurt much, instead it would prevent this problem from occurring again ...